tests/openvino/test_exporters_cli.py

```diff
@@ -118,21 +118,23 @@ def test_exporters_cli(self, task: str, model_type: str):
         for arch in SUPPORTED_ARCHITECTURES
         if not arch[0].endswith("-with-past") and not arch[1].endswith("-refiner")
     )
-    @unittest.skipIf(not is_openvino_tokenizers_available(), reason="OpenVINO Tokenizers not available")
     def test_exporters_cli_tokenizers(self, task: str, model_type: str):
         with TemporaryDirectory() as tmpdir:
             output = subprocess.check_output(
```

Comment:
we should also check whether no error was raised here

Reply:
With
```bash
optimum-cli export openvino --model hf-internal-testing/tiny-random-t5 --task text2text-generation ov_model
```

I'm getting the following error : 
```
OpenVINO Tokenizer export for T5TokenizerFast is not supported. Exception: [Errno 2] No such file or directory: '/tmp/tmprj8zsg44/spiece.model'
```

Reply:
Would like to have this fixed before making the tokenizer export default / merging this PR

Reply:
It is not a bug, we don't support the Unigram model from `tokenizer.json` yet, only from sentencepiece model file. If you use a [repository](https://huggingface.co/google-t5/t5-base/tree/main) that has such a file, the tokenizer will be converted.

I added one more check for the vocab file here: https://github.com/openvinotoolkit/openvino_tokenizers/pull/116
This will transform
```shell
OpenVINO Tokenizer export for T5TokenizerFast is not supported. Exception: [Errno 2] No such file or directory: '/tmp/tmprj8zsg44/spiece.model'
```
into:
```shell
OpenVINO Tokenizer export for T5TokenizerFast is not supported. Exception: Cannot convert tokenizer of this type without `.model` file.
```

The issue is that the original tokenizer has info about `.model` file in the tokenizer object, but it does not have it in practice:
<img width="323" alt="image" src="https://github.com/huggingface/optimum-intel/assets/51917466/2fd5f24a-3325-448f-aacb-72292002c88f">

I would argue that this is an edge case of a test repo and most tokenizers with info about the `.model` file have it, so it should not block the merge.

Reply:
It doesn't throw an error in this case, just prints an error message from a tokenizer conversion process. The model is converted fine:
```bash
>> optimum-cli export openvino --model hf-internal-testing/tiny-random-t5 --task text2text-generation openvino_model
INFO:nncf:NNCF initialized successfully. Supported frameworks detected: torch, onnx, openvino
Framework not specified. Using pt to export the model.
The task `text2text-generation` was manually specified, and past key values will not be reused in the decoding. if needed, please pass `--task text2text-generation-with-past` to export using the past key values.
Using the export variant default. Available variants are:
    - default: The default ONNX variant.
Using framework PyTorch: 2.1.2+cu121
Overriding 1 configuration item(s)
        - use_cache -> False
Using framework PyTorch: 2.1.2+cu121
Overriding 1 configuration item(s)
        - use_cache -> False
/home/apaniuko/python/optimum-intel/venv/lib/python3.10/site-packages/transformers/modeling_utils.py:908: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  if causal_mask.shape[1] < attention_mask.shape[1]:
OpenVINO Tokenizer export for T5TokenizerFast is not supported. Exception: [Errno 2] No such file or directory: '/tmp/tmpyww_b5i_/spiece.model'

>> ls openvino_model/
config.json             openvino_decoder_model.bin  openvino_encoder_model.bin  special_tokens_map.json  tokenizer.json
generation_config.json  openvino_decoder_model.xml  openvino_encoder_model.xml  tokenizer_config.json
```

The confusion comes from the fact that this is the last message from the conversion process. Maybe adding the "The model converted successfully" message will be beneficial. This message can be added in a separate PR as well.

Reply:
Can we either disable this warning or disable export for unsupported cases ?

Reply:
This warning is for any exception that appeared during conversion. After seeing this message, the user can create an issue for a particular model/tokenizer support.
I also prefer to tell the user that the tokenizer is not converted, rather than silently not including it without warning, so the lack of a (de)tokenizer model won't be a surprise.

Reply:
I'd agree if the export of the tokenizer was an explicit choice of the user (current `--convert-tokenizer`) but as this PR makes it default I think having such warning can be an issue as it makes it look like the export failed

Reply:
I changed the log level to debug, the message won't be visible by default.

Reply:
thanks @apaniukov